base/src/main/java/com/austindroids/austinfeedsme/eventsfilter/EventFilterAdapter.java

```diff
@@ -70,40 +65,11 @@ public void onBindViewHolder(EventFilterAdapter.ViewHolder viewHolder, int posit
     }
 
     public void addEvents(List<Event> events) {
-        removeDuplicateEvents(events);
-
+        this.events.clear();
```

Comment:
Consider using `ListAdapter` from AndroidX so that you don't need to do this sort of magic, and just pass in an unmodifiable collection instead.